examples/fungible/src/contract.rs

```diff
@@ -60,8 +75,12 @@ impl Contract for FungibleToken {
                 amount,
                 target_account,
             } => {
-                Self::check_account_authentication(None, runtime.authenticated_signer(), owner)?;
-                self.debit(owner, amount).await?;
+                Self::check_account_authentication(
```

Comment:
Maybe a non-static method `self.check_account_authentication(..)` is better now

Reply:
Done, good idea!